### DISCUSSION TITLE
docker 仓库添加错误

### DISCUSSION THREAD

**Discussion by Shuai-S** [1Panel-dev/1Panel#5480] [Q&A]
### 1Panel 版本

v1.10.10-lts

### 问题描述

添加docker仓库，直接报错，然后docker直接关闭，无法启动

<img width="765" alt="image" src="https://github.com/1Panel-dev/1Panel/assets/52077275/659015fe-b82f-414d-b979-de46abb990ac">

<img width="738" alt="image" src="https://github.com/1Panel-dev/1Panel/assets/52077275/0dd47d1e-3d5b-48e2-899b-729cc55a233d">

### 重现步骤

<img width="765" alt="image" src="https://github.com/1Panel-dev/1Panel/assets/52077275/659015fe-b82f-414d-b979-de46abb990ac">

**Comment by Shuai-S** [comment]
重启服务器 1panel 无法启动，docker报错了
<img width="1021" alt="image" src="https://github.com/1Panel-dev/1Panel/assets/52077275/319b7ecc-3cd8-4fce-b7a9-e4da103bcdd2">

**Reply by wanghe-fit2cloud** [reply]
截图看一下 daemon.json 文件内容：
```shell
cat /etc/docker/daemon.json
```

**Reply by Shuai-S** [reply]
<img width="1008" alt="image" src="https://github.com/1Panel-dev/1Panel/assets/52077275/c45a469c-1132-4f1d-be1e-35fa16701868">
<img width="1006" alt="image" src="https://github.com/1Panel-dev/1Panel/assets/52077275/645f6afc-c083-42d3-9431-4d9f1c989e98">
我已经解决了，把insecure-registries 删除，然后重启docker就行了
但是只要修改和docker仓库相关的数据，都会出现这个错误，bao包括设置私有仓库
<img width="772" alt="image" src="https://github.com/1Panel-dev/1Panel/assets/52077275/477e713d-9c5f-4b10-9590-3d861279b757">
